frontend/src/components/styles/layout/_search.scss

```diff
@@ -0,0 +1,72 @@
+.search {
+    position: relative;
+    padding: 3em 0em;
```

Comment:
although I would probably remove the padding

Reply:
done :)